src/locale/en.json

```diff
@@ -156,6 +156,8 @@
   },
   "FashionDrawer": {
     "Accept": "Save fashion",
+    "CannotFitOrnament": "This item cannot fit an ornament",
```

Comment:
Maybe more clearly "This item does not have an ornament socket"?

Reply:
I wasn't sure if the user base would know what a "socket" is but I am probably over thinking it. Will change.